src/base-app.js.flow

```diff
@@ -0,0 +1,38 @@
+// @flow
```

Comment:
Awesome!  Still not fully ideal, but I like that we can get rid of the `libdef` and not have to maintain separate types internally and externally.